components/navbar/navbar.jsx

```diff
@@ -23,7 +23,7 @@ class NavBar extends React.Component {
         <div className="container">
           <div className="row">
             <div className="logo col-md-4 mb-3 mb-sm-0">
-              <IndexLink to="/" className="d-inline-block"><img src="/assets/svg/pulse-wordmark.svg" width="204" height="34" alt="Mozilla Pulse" /></IndexLink>
+              <li className="d-inline-block"><img src="/assets/svg/pulse-wordmark.svg" width="204" height="34" alt="Mozilla Pulse" /></li>
```

Comment:
`<li>` are list items, they only go inside lists (ordered, `<ol>`, or unordered, `<ul>`). In this case we have `<div><IndexLink><img></IndexLink></div>` so just remove the surrounding `<IndexLink>`, or if we need that `d-inline-block` class applied, we can see if we can apply it directly to the parent div, or to the img. If both of those look wrong, then instead of removing it entirely, turn it into a `<div className="d-inline-block"> <img ..../> </div>`